src/main/sensors/adcinternal.c

```diff
@@ -86,12 +87,15 @@ void adcInternalProcess(timeUs_t currentTimeUs)
     adcVrefintValue = updateMovingAverageUint16(&adcVrefintAverageState, vrefintSample);
     adcTempsensorValue = updateMovingAverageUint16(&adcTempsensorAverageState, tempsensorSample);
 
-    int32_t adcTempsensorAdjusted = (int32_t)adcTempsensorValue * 3300 / getVrefMv();
+    vrefMv = 3300 * adcVrefintValue / adcVREFINTCAL;
+
+    int32_t adcTempsensorAdjusted = (int32_t)(adcTempsensorValue * 3300) / vrefMv;
     coreTemperature = ((adcTempsensorAdjusted - adcTSCAL1) * adcTSSlopeK + 30 * 1000 + 500) / 1000;
 
     DEBUG_SET(DEBUG_ADC_INTERNAL, 0, coreTemperature);
-    DEBUG_SET(DEBUG_ADC_INTERNAL, 1, vrefintSample);
-    DEBUG_SET(DEBUG_ADC_INTERNAL, 2, tempsensorSample);
+    DEBUG_SET(DEBUG_ADC_INTERNAL, 1, vrefMv);
```

Comment:
No point in switching the existing order of debug values around, and making any instructions relying on the order void. `vrefMv` should go at the end.

Reply:
I did wonder wonder about that, my reasoning was: outputs first, related stuff afterwards.  I'll move it, you can re-order it later if you like.

Reply:
While I am all for well-ordered approaches, I think in cases where an external facing API is concerned, avoiding change to the existing API is more important, and as such appending of new items at the end is preferrable.

Reply:
done